client/lib/post-normalizer/rule-content-link-jetpack-carousels.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External Dependencies
+ */
+import { forEach, get } from 'lodash';
+
+export default function linkJetpackCarousels( post, dom ) {
+	const galleries = dom.querySelectorAll( '.tiled-gallery' );
+
+	forEach( galleries, gallery => {
+		let extra = get( gallery, [ 'dataset', 'carouselExtra' ], false );
```

Comment:
so a gallery carousel can have either a property: `dataset.courselExtra` or a property `data-carousel-extra`?
What creates that property / when does one exist vs. the other?